src/icalendar/cal.py

```diff
@@ -587,6 +588,74 @@ def is_thunderbird(self) -> bool:
         return any(attr.startswith("X-MOZ-") for attr in self.keys())
 
 
+    @classmethod
+    def from_file(cls, file: Union[str, Path], multiple: bool = False):
+        """Create a Component from a file.
+
+        This class method can be used by any Component subclass (Calendar, Event, etc.)
+        to read their data from a file.
+
+        Args:
+            file: The file to read from. Can be:
+                - A string path to a file
+                - A Path object
+            multiple: If True, allows parsing multiple components from the file.
+
+        Returns:
+            If multiple=False (default):
+                A single Component instance of the appropriate type
+            If multiple=True:
+                A list of Component instances
+
+        Raises:
+            FileNotFoundError: If the file path doesn't exist
+            ValueError: If the file contents are not valid iCalendar format
+
+        Example:
+            >>> from icalendar import Calendar
+            >>> # Read a calendar file
+            >>> cal = Calendar.from_file("src/icalendar/tests/calendars/example.ics")
+            >>> # Read multiple calendars
```

Comment:
You can remove the >>> here

Reply:
Are you suggestion to remove the `>>>` on the lines where there is a comment? or perhaps just remove the comments altogether?

Reply:
https://github.com/collective/icalendar/pull/757/files/804ab5297092428c2ec0c440f28d50db5ac686ba#r1878247067

Reply:
if I remove the `>>>` from the comments then the test fails
This doesn't work:
```
        Example:
            >>> from icalendar import Calendar
            >>> from pathlib import Path
            # Read a calendar file
            >>> cal = Calendar.from_file("src/icalendar/tests/calendars/example.ics")
            # Read multiple calendars
            >>> cals = Calendar.from_file("src/icalendar/tests/calendars/multiple_calendar_components.ics", multiple=True)
            # or pass a Path object
            >>> path = Path("src/icalendar/tests/calendars/example.ics")
            >>> cal = Calendar.from_file(path)
```
This does work:
```
        Example:
            >>> from icalendar import Calendar
            >>> from pathlib import Path
            >>> # Read a calendar file
            >>> cal = Calendar.from_file("src/icalendar/tests/calendars/example.ics")
            >>> # Read multiple calendars
            >>> cals = Calendar.from_file("src/icalendar/tests/calendars/multiple_calendar_components.ics", multiple=True)
            >>> # or pass a Path object
            >>> path = Path("src/icalendar/tests/calendars/example.ics")
            >>> cal = Calendar.from_file(path)
```

Reply:
It needs a new line:


```
>>> from pathlib import Path

# Read a calendar file
```

Otherwise the comment is considered output.